### PR TITLE
Add content from Curtain Call Player's Guide

### PR DESCRIPTION
**Pull request by TikaelSol**
Also adds illusion trait to equipment and weapons

### CODE REVIEW

File: packs/backgrounds/always-chosen-last.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Streetwise",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Streetwise"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/always-chosen-last.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/amateur-director.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Group Impression",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Group Impression"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/amateur-director.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/art-tutor.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Crafter's Appraisal",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Crafter's Appraisal"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/art-tutor.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/bully-or-baiter.json [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Intimidating Glare",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Intimidating Glare"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/bully-or-baiter.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/sideshow-scion.json [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Acrobatic Performer",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Acrobatic Performer"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/sideshow-scion.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/supportive-sponsor.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Battle Medicine",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Battle Medicine"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/supportive-sponsor.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/town-troublemaker.json [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+        },
+        "items": {},
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/town-troublemaker.json:31]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/wandering-libertine.json [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+                "img": "icons/sundries/books/book-red-exclamation.webp",
+                "level": 1,
+                "name": "Charming Liar",
+                "uuid": "Compendium.pf2e.feats-srd.Item.Charming Liar"
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/backgrounds/wandering-libertine.json:38]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/campaign-effects/persona-trait.json [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+        "level": {
+            "value": 1
+        },
+        "prerequisites": {
+            "value": []
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/campaign-effects/persona-trait.json:25]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/deities/the-curtain-call.json [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+                "passion"
+            ]
+        },
+        "font": [
+            "heal"
+        ],
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Curtain Call Player's Guide"
```
Comment (simonward) [inline packs/deities/the-curtain-call.json:31]:
```suggestion
            "title": "Pathfinder: Curtain Call Player's Guide"
```

File: packs/backgrounds/always-chosen-last.json (hunk 1 above)
Comment (TikaelSol) [reply packs/backgrounds/always-chosen-last.json:38]: We're going to flip to no colon being the standard, since that's what Paizo publishes them with

**Pull request merged**